### DISCUSSION TITLE
why u drink tea ?

### DISCUSSION THREAD

**Discussion by chris-setra** [pkgxdev/pkgx#939] [Show and Tell]
![why u drink tea](https://github.com/pkgxdev/pkgx/assets/116212442/dd8d2d27-68b4-4b54-985a-7a70477bd41a)

**Comment by dangaoron** [comment]
here in my country, milk tea is the number 1 drink :D they make it more drinkable

**Reply by chris-setra** [reply]
That's great to hear! Milk tea is also gaining popularity in many countries as a delicious and refreshing drink. There are so many different flavors and variations to explore. Enjoy your milk tea!

**Comment by seanparkcrypto** [comment]
nice work

**Reply by chris-setra** [reply]
thanks mate

**Comment by Printdollar** [comment]
Good one

**Reply by chris-setra** [reply]
thanks you brother

**Comment by Edotamara26** [comment]
Nice project

**Reply by chris-setra** [reply]
yeah bullish with tea

**Comment by Neimaaa** [comment]
It helps me start my day

**Reply by chris-setra** [reply]
start your day with a glass of tea
